### ISSUE TITLE
Scenarios for console arcs

### ISSUE THREAD

**Issue by Grelo** [07th-mod/higurashi-patch-compiler#42]
Someutsushi:

![scenario_a](https://user-images.githubusercontent.com/22790067/40177066-bb97fd6a-59dd-11e8-9bb1-8a91853a9b5f.png)
![scenario_b](https://user-images.githubusercontent.com/22790067/40177067-bbb5aaf4-59dd-11e8-867e-7acb1c7c0e75.png)
![scenario_c](https://user-images.githubusercontent.com/22790067/40177068-bbd0ffc0-59dd-11e8-93ea-4109cfcd3c1a.png)

Kageboushi:

![scenario_a](https://user-images.githubusercontent.com/22790067/40177254-52934ba2-59de-11e8-9e2a-702697280bbf.png)
![scenario_b](https://user-images.githubusercontent.com/22790067/40177126-ec49b41c-59dd-11e8-94e6-58ecd0f2e23e.png)
![scenario_c](https://user-images.githubusercontent.com/22790067/40177127-ec655a8c-59dd-11e8-84ac-ae8917d3a852.png)

Tsukiotoshi:

![scenario_a](https://user-images.githubusercontent.com/22790067/40177289-6b177946-59de-11e8-8710-670aeda7e2d3.png)
![scenario_b](https://user-images.githubusercontent.com/22790067/40177291-6b37492e-59de-11e8-8324-d0c15b10ca83.png)
![scenario_c](https://user-images.githubusercontent.com/22790067/40177294-6b51914e-59de-11e8-8022-d349095d1381.png)

**Comment by enumag** [comment]
@DoctorDiablo Are these already included in the console arcs? Could you give us a good background and the text for Taraimawashi?

**Comment by DoctorDiablo** [comment]
Yeah, I'll need one more of these for Taraimawashi.  @Grelo can you make one with this background and text:

>Taraimawashi
>-Idly Spinning-
>
>This chapter is an alternate take on Onikakushi.
>It can be read after completion of Chapter 3: Tatarigoroshi.
>
>Keiichi does his best to avoid the the major conflicts in Hinamizawa.

![y_kawa3](https://user-images.githubusercontent.com/20390474/43029890-8d4abf4e-8c58-11e8-975a-7598a0727b76.png)

**Comment by Grelo** [comment]
@DoctorDiablo Ok here it is :)

Taraimawashi:
![scenario_a](https://user-images.githubusercontent.com/22790067/43322703-b67da2c6-91af-11e8-8143-2b455e51a1c5.png)
![scenario_b](https://user-images.githubusercontent.com/22790067/43322704-b69a93d6-91af-11e8-8ab4-9e92924de795.png)
![scenario_c](https://user-images.githubusercontent.com/22790067/43322705-b6b688ac-91af-11e8-855c-04c8ae179867.png)

**Comment by enumag** [comment]
In the main arcs we've updated scenarios to use separate images and layers like [this](https://github.com/07th-mod/tsumihoroboshi/blob/master/Update/flow.txt#L89-L134) and added a screen to show the mod hotkeys. I'd like to do this in console arcs as well. I'll need the images with arc names and descriptions to do that. In the main arcs they are named:

```
scenario/title
scenario/title_j
scenario/description
scenario/description_j
```
